libfwupd/fwupd-remote-private.h

```diff
@@ -48,6 +48,14 @@ fwupd_remote_set_username(FwupdRemote *self, const gchar *username) G_GNUC_NON_N
 void
 fwupd_remote_set_password(FwupdRemote *self, const gchar *password) G_GNUC_NON_NULL(1);
 void
+fwupd_remote_set_auth_type(FwupdRemote *self, FwupdRemoteAuthType auth_type) G_GNUC_NON_NULL(1);
+void
+fwupd_remote_set_aws_access_key(FwupdRemote *self, const gchar *access_key) G_GNUC_NON_NULL(1);
+void
+fwupd_remote_set_aws_secret_key(FwupdRemote *self, const gchar *secret_key) G_GNUC_NON_NULL(1);
```

Comment:
@superm1 is this different to the username/password?

Reply:
is *aws* the correct prefix here? I know it's what most people are using, but isn't this really a "s3-compatible" authentication rather then something Amazon-specific?